File: app/src/main/java/com/example/myworkout/MainActivity.java
package com.example.myworkout;

import android.support.annotation.NonNull;

import androidx.core.widget.DrawerLayout;
import android.support.v7.app.ActionBarDrawerToggle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

//public class MainActivity extends AppCompatActivity implements AlertStartExercise.sendExercise {
//
//    @Override
//    protected void onCreate(Bundle savedInstanceState) {
//        super.onCreate(savedInstanceState);
//        setContentView(R.layout.activity_main);
//
//
//        Toolbar toolbar = findViewById(R.id.toolbar);
//        setSupportActionBar(toolbar);
//
//        BottomNavigationView navigationView = findViewById(R.id.nav_view);
//        navigationView.setOnNavigationItemSelectedListener(navListener);
//
//        getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container, new MyWorkoutsFragment()).commit();
//
//    }
//
//    private BottomNavigationView.OnNavigationItemSelectedListener navListener = new BottomNavigationView.OnNavigationItemSelectedListener() {
//        @Override
//        public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
//            switch (menuItem.getItemId()) {
//                case R.id.nav_workout_builder:
//                    getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container, new WorkoutBuilderFragment()).commit();
//                    break;
//                case R.id.nav_my_workouts:
//                    getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container, new MyWorkoutsFragment()).commit();
//                    break;
//
//                case R.id.nav_workout_history:
//                    getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container, new WorkoutHistoryFragment()).commit();
//            }
//
//            return true;
//        }
//    };
//
//    @Override
//    public void exerciseComplete(Exercise exercise) {
//
//    }
//}

File: app/src/main/java/com/example/myworkout/MainActivity.kt
package com.example.myworkout

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar

import com.example.myworkout.AlertStartExercise.sendExercise
import com.google.android.material.bottomnavigation.BottomNavigationView


class MainActivity : AppCompatActivity(), sendExercise {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val toolbar = findViewById<Toolbar>(R.id.toolbar)
        setSupportActionBar(toolbar)
        val navigationView = findViewById<BottomNavigationView>(R.id.nav_view)
        navigationView.setOnNavigationItemSelectedListener(navListener)
        supportFragmentManager.beginTransaction().replace(R.id.fragment_container, MyWorkoutsFragment()).commit()
    }

    private val navListener = BottomNavigationView.OnNavigationItemSelectedListener { menuItem ->
        when (menuItem.itemId) {
            R.id.nav_workout_builder -> supportFragmentManager.beginTransaction().replace(R.id.fragment_container, WorkoutBuilderFragment()).commit()
            R.id.nav_my_workouts -> supportFragmentManager.beginTransaction().replace(R.id.fragment_container, MyWorkoutsFragment()).commit()
            R.id.nav_workout_videos -> supportFragmentManager.beginTransaction().replace(R.id.fragment_container, WorkoutVideosFragment()).commit()
            R.id.nav_workout_history -> supportFragmentManager.beginTransaction().replace(R.id.fragment_container, WorkoutHistoryFragment()).commit()
        }
        true
    }

    override fun exerciseComplete(exercise: Exercise) {}
}
